ui_framework/components/prompt_for_items/prompt_for_items.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import { KuiButtonIcon, KuiLinkButton } from '../button';
+import { KuiPromptForItemsMessage } from  './prompt_for_items_message';
+
+export function KuiPromptForItems({ itemType, promptMessage, promptButtonText, addHref, className, ...rest }) {
+  const classes = classnames('kuiPromptForItems', className);
+  return (
+    <div className={classes} {...rest} >
+      <KuiPromptForItemsMessage>
+        {promptMessage}
+      </KuiPromptForItemsMessage>
+
+      <div className="kuiPromptForItems__actions">
+        <KuiLinkButton
+          icon={<KuiButtonIcon type="create" />}
+          aria-label={`Add a new ${itemType}`}
+          className="testClass1 testClass2"
+          data-test-subj={`addNewPromptButton`}
+          buttonType="primary"
+          href={addHref}
+        >
```

Comment:
If we did that, should it still be called PromptForItems?  At that point it's just a label on top of an action component.  My concern with making everything so generic is that then it's not as easy to know what to use in which situations. I don't really feel strongly either way though, so if you think this is the way to go I can make the change.  Another option is to wait until we require a prompt with more/different buttons.